images/cert-manager/config/main.tf

```diff
@@ -0,0 +1,28 @@
+variable "name" {
+  description = "Package name (e.g. cainjector, acmeresolver, controller, webhook)"
+}
+
+variable "suffix" {
+  description = "Package name suffix (e.g. version stream)"
+  default     = ""
+}
+
+variable "extra_packages" {
+  description = "Additional packages to install."
+  type        = list(string)
+  default     = []
+}
+
+module "accts" { source = "../../../tflib/accts" }
+
+output "config" {
+  value = jsonencode({
```

Comment:
Are we potentially double-encoding? is this a problem? 🤔 

```terraform
module "latest-dev" {
  config            = jsonencode(module.latest[each.key].config)
  ...
}
```

Reply:
This is a different config!

This line is the config that comes out of the non-dev version to ensure that everything is "locked" to the same packages in dev, except for anything new we add on top.